MeNext/Layout/MainPage.cs

```diff
@@ -24,18 +24,22 @@ public MainPage()
 
 			var homeScreen = new NavigationPage(new HomeScreen());
 			homeScreen.Title = "Home";
+			//homeScreen.Icon = "homeScreenIcon.png";  If we make this icon
```

Comment:
Does this work? If so any reason not to leave it in?

@samsaksfithian @daniel-centore thoughts?